app/src/main/java/com/example/fittracker/mvp/view/UserSettingsView.java

```diff
@@ -34,33 +36,66 @@ public int hashCode() {
     }
 
     @Override
-    public void onEmailFocusChange() {
-
+    public void onEmailClick() {
+        mailView.setVisibility(View.GONE);
+        mailEdit.setVisibility(View.VISIBLE);
+        resetName();
+        resetPassword();
```

Comment:
why when you click on the email field you set the visibility of the password?

if this is necessary this must be in the presenter

Reply:
Done